core/src/nucleus/ribosome/api/get.rs

```diff
@@ -87,7 +90,7 @@ mod tests {
     #[test]
     /// test that we can round trip bytes through a get action and it comes back from wasm
     fn test_get_round_trip() {
-        let (runtime, _) = test_zome_api_function_runtime("get", test_args_bytes());
+        let (runtime, _) = test_zome_api_function_runtime("hc_get_entry", test_args_bytes());
```

Comment:
nice, i've been heading in this direction on the hash table side of things too

a flat "get" is just not that helpful when we have so many similar things floating around >.<